### DISCUSSION TITLE
Add an icon to suggest "draggable" action

### DISCUSSION THREAD

**Discussion by surajtalreja1** [tailwindlabs/heroicons#620] [Ideas]
This is typically represented with a double vertical ellipse

**Comment by mattfelten** [comment]
Yes please. I was just looking for this.

**Comment by mattfelten** [comment]
I made one by doubling `ellipsis-vertical`.

![CleanShot 2023-01-03 at 20 28 02](https://user-images.githubusercontent.com/488744/210484694-8510b18d-1be1-4686-a69b-86831ca2149a.png)

Download Links:
[ellipsis-double-vertical-20.svg](https://user-images.githubusercontent.com/488744/210484710-b8794637-5d11-47c7-9cdf-9d2837548fb1.svg)
[ellipsis-double-vertical-24.svg](https://user-images.githubusercontent.com/488744/210484711-c49ec478-e89f-456f-a311-c92d450dd0f2.svg)

**Reply by mattfelten** [reply]
Happy to make a PR out of this if the team is open to it

**Comment by ndri** [comment]
I added `dots-2x3` and `dots-3x2` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: 
- https://sidekickicons.com/?icon=dots-2x3
- https://sidekickicons.com/?icon=dots-3x2
